src/components/users/login/LoginPage.js

```diff
@@ -21,6 +23,7 @@ const LoginPage = () => {
       .post('/login', user)
       .then((res) => {
         localStorage.setItem(process.env.REACT_APP_USER_TOKEN, res.data.data.token);
+        setLoggedInUser(res.data.data.user)
```

Comment:
you might want to store the user data in localStorage as well? What happens when we load the logged in user from local storage instead of calling the LOGIN endpoint?